tests/phpunit/tests/formatting/SanitizeFileName.php

```diff
@@ -11,10 +11,11 @@ function test_munges_extensions() {
 	}
 
 	function test_removes_special_chars() {
-		$special_chars = array("?", "[", "]", "/", "\\", "=", "<", ">", ":", ";", ",", "'", "\"", "&", "$", "#", "*", "(", ")", "|", "~", "`", "!", "{", "}", "%", "+", chr(0));
-		$string = 'test';
-		foreach ( $special_chars as $char )
+		$special_chars = array( '?', '[', ']', '/', '\\', '=', '<', '>', ':', ';', ',', "'", '"', '&', '$', '#', '*', '(', ')', '|', '~', '`', '!', '{', '}', '%', '+', chr( 0 ) );
```

Comment:
The backslash seems wrong. And I hate to say it, but should this array be expanded?

Reply:
It's a normal array, not associative, so it doesn't need expanding. 
But the backslash is still wrong.

Reply:
Single line seems fine and upstream is single quotes:
https://github.com/WordPress/wordpress-develop/blob/trunk/tests/phpunit/tests/formatting/sanitizeFileName.php

Reply:
Both are wrong.

Reply:
I think I was misreading it. My mistake.